### ISSUE TITLE
Missing geary icons

### ISSUE THREAD

**Issue by bil-elmoussaoui** [numixproject/numix-icon-theme#768]
Hello,
There's still some missing icons in Geary mail client.

- [ ] tag
- [ ] tags

Symbolic icons : 
- [ ] accessories-text-editor-symbolic
- [ ] application-menu-symbolic
- [ ]
- [ ]
- [ ]

**Comment by wa4557** [comment]
Do you have a screenshot of that (how do for example tags and tag differ?)

**Comment by bil-elmoussaoui** [comment]
@wa4557 I don't have those icons in Ubuntu Gnome! :S

**Comment by palob** [comment]
So they're only used in the elementary UI?

**Comment by bil-elmoussaoui** [comment]
Non! The version of geary shipped in Ubuntu Gnome uses symbolic icons

**Comment by palob** [comment]
Ah OK, so that's the normal headerbar icons.

**Comment by bil-elmoussaoui** [comment]
Yes, i will add the symbolic icons too

**Comment by bil-elmoussaoui** [comment]
@wa4557 tags icon does not exists in the default elementary theme! maybe just having multiple tag icons? 
default tag icon in elementary
![Uploading Screenshot from 2015-10-31 00:01:13.png…]()

**Comment by bil-elmoussaoui** [comment]
I updated this one! i added Geary Gnome icons (uses only symbolic ones)

**Comment by palob** [comment]
Is ``tags`` the same as ``tag`` visually?

**Comment by bil-elmoussaoui** [comment]
Yes

**Comment by palob** [comment]
Hmm, so we can't just use the existing design from ``tag-new``, make it bigger and align it.

**Comment by bil-elmoussaoui** [comment]
@palob You mean we can use the `tag-new` design ?

**Comment by palob** [comment]
After a bit of tweaking. But no, because it's white.

There's already the issue with for instance ``document-import`` showing up as arrow only in some elementary apps' welcome screens when using the elmentary GTK theme.
